pokemon_battle_arena/src/db/enums.rs

```diff
@@ -70,56 +69,56 @@ pub enum Ailment {
     Torment,
     Disable,
     Yawn,
-    Heal_Block,
-    No_Type_Immunity,
-    Leech_Seed,
+    HealBlock,
+    NoTypeImmunity,
+    LeechSeed,
     Embargo,
-    Perish_Song,
+    PerishSong,
     Ingrain,
 }
 
 ///All the major status Changes that can not be caused at the same time.
 #[derive(Debug, Clone, PartialEq)]
-pub enum Non_Volatile {
+pub enum NonVolatile {
     Undefined,
     Paralysis,
     Sleep,
     Freeze,
     Burn,
     Poison,
-    Bad_Poison,
+    BadPoison,
 }
 
 ///Flags that have a influence at the end of each turn.
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
-pub enum End_Of_Turn {
+pub enum EndOfTurn {
     //absorbs some HP at the End of every Turn
-    Leech_Seed,
+    LeechSeed,
     //Counts from 0 to 4, one step each round, even in the turn it was initially used.
     //When 4 is reached the Pokemon faints. Counting only continues when the Pokemon is part of
     //the battle, but the counter will not be resetted if the Pokemon is changed.
```

Comment:
Doc comments need to start with three slashes: `///`. Also: please add a space between the slashes and the text:

    /// Counts from ...
    /// When 4 is reached...